docs/deployment_patterns/how_to_instantiate_a_data_context_hosted_environments.md

```diff
@@ -4,11 +4,11 @@ title: Deploying Great Expectations in a hosted environment without file system
 
 If you follow the steps of the [Getting Started](../tutorials/getting_started/intro) tutorial, you create a standard deployment of Great Expectations. By default, this relies on two components:
 
-#. The Great Expectations [CLI](../guides/miscellaneous/how_to_use_the_great_expectations_cli) to initialize a Data Context, create Expectation Suites, add Datasources, etc.
-#. The ``great_expectations.yml`` file to configure your Data Context, e.g. to point at different Stores for Validation Results, etc.
+1. The Great Expectations [CLI](../guides/miscellaneous/how_to_use_the_great_expectations_cli) to initialize a Data Context, create Expectation Suites, add Datasources, etc.
+2. The ``great_expectations.yml`` file to configure your Data Context, e.g. to point at different Stores for Validation Results, etc.
 
 
-However, you might not have these components available in hosted environments, such as [Databricks](./how_to_instantiate_a_data_context_on_databricks_spark_cluster), [AWS EMR](./how_to_instantiate_a_data_context_on_an_emr_spark_cluster), Google Cloud Composer, and others. This workflow guide will outline the main steps required to successfully use Great Expectations in a hosted environment.
+However, you might not have these components available in hosted environments, such as Databricks, AWS EMR, Google Cloud Composer, and others. This workflow guide will outline the main steps required to successfully use Great Expectations in a hosted environment.
```

Comment:
I removed these links because they are linked again in the lines immediately below.